app/scripts/components/executions/execution-status.js

```diff
@@ -135,6 +135,7 @@ var ExecutionStatus = React.createClass({
       <section className='page__section'>
         <div className='heading__wrapper--border'>
           <h2 className='heading--medium with-description'>Events</h2>
+          <p><b>More details</b> includes details of the step function event execution, including the tasks' name and version. To find all task name and versions, select <b>More details</b> for the last Lambda- or Activity-type event. There you should find a key / value pair "workflow_tasks" which lists all tasks' version, name and arn.</p>
```

Comment:
Since this depends on Cumulus 1.9.1+ and the appropriate message adapters, maybe add a note about 'If task/version tracking is enabled (...)'. Also, it is worth remembering that `workflow_tasks` is only updated as each task executes, so only the last output will have information about all the tasks that have run.

Reply:
Good point about the version,
And you do say the last lambda should have them all so that's fine, it just a little difficult to find. What if we pulled it out somewhere?

Reply:
@mennovandiermen good point, added a note here: https://github.com/nasa/cumulus-dashboard/pull/485/commits/0f0c09253bd90315d8dac37929ebd0e5a314b572

Reply:
Maybe I'm misunderstanding but I did try and describe that the _final_ event should be inspected:

> To find all task name and versions, select <b>More details</b> for the last Lambda- or Activity-type event.

I removed the sentence that came before this sentence to make the instruction more obvious - let me know if you think this content will be sufficient in helping users find the information they need.